icssc/AntAlmanac
# Add ability to name, add, and delete schedules

## Summary
- Users can add any number of schedules they want with their own names. They can rename and delete these schedules.
- Users that already have saved schedules will start with 4 schedules (Schedule 1, Schedule 2, Schedule 3, Schedule 4).
- New users that do not have any saved schedules will start with just one schedule named `Schedule 1`.

## Test Plan
- Use the test plans from #350 and #368, and see if they pass.
- Make sure new users and existing users can properly save and load their schedules.

## Issues
- Closes #330 
- Closes #331 
- Closes #332 
- Closes #333
